crates/cribo/debug_resolver.py

```
@@ -0,0 +1,16 @@
+import sys
+import os
+
+# Test what the resolver should find
+test_dir = "tests/fixtures/stickytape_test_scripts/script_using_from_to_import_module"
+
+print("Checking directory structure:")
+print(f"greetings/ exists: {os.path.exists(os.path.join(test_dir, 'greetings'))}")
+print(f"greetings/__init__.py exists: {os.path.exists(os.path.join(test_dir, 'greetings/__init__.py'))}")
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)